src/diffusers/models/transformers/transformer_qwenimage.py

```diff
@@ -219,6 +219,7 @@ def forward(self, video_fhw, txt_seq_lens, device):
                 video_freq = self.rope_cache[rope_key]
             else:
                 video_freq = self._compute_video_freqs(frame, height, width, idx)
+            video_freq = video_freq.to(device)
```

Comment:
Without this, the `test_to_device` test fails:
https://github.com/huggingface/diffusers/blob/e682af202787c44da4c7e583b95ec7f42dc45029/tests/pipelines/test_pipelines_common.py#L1503